File: java/Silly.java
public class Silly {
    private int _x;
    private int _y;
    public Silly(int x,int y) {
        _x = x;
        _y = y;
    }
    long callMe(long z) {
        long r = (long)_x * z + _y;
        return r;
    }
}

File: Clo/Clo/Silly.m
/************************************************************************
 Mozilla Public License
 
 Copyright (c) 2012 NICTA, Laurent Michel and Pascal Van Hentenryck

 This Source Code Form is subject to the terms of the Mozilla Public
 License, v. 2.0. If a copy of the MPL was not distributed with this
 file, You can obtain one at http://mozilla.org/MPL/2.0/.

 ***********************************************************************/

#import "Silly.h"


@implementation Silly

- (id)init:(CPInt)x y:(CPInt)y;
{
    self = [super init];
    _x = x;
    _y = y;
    return self;
}

- (void)dealloc
{
    [super dealloc];
}

-(long)callMe:(long) z 
{
   long r =  (long)_x * z + _y;
   return r;
}

@end
